src/styles/blocks/nav.scss

```
@@ -0,0 +1,65 @@
+
+.nav {
+  &__list {
+    display: flex;
+    align-items: center;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    display: block;
+    padding: 23px 0;
+    color: #060b35;
+    text-decoration: none;
+    align-content: center;
+    transition: all 0.3s;
+
+    &:hover {
+      color: #{$color-accent};
```

why you need this?
use it in all cases
```suggestion
      color: $color-accent;
```